tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -400,8 +403,9 @@ public class LinkModeFullClass {
 			}, logcatPath, 90), "Linker test app did not run successfully.");
 
 			var logcatOutput = File.ReadAllText (logcatPath);
+			var logcatOutputCut = Substring(Math.Min(logcatOutput.Length - 3000, logcatOutput), logcatOutput.Length);
```

Comment:
This doesn't build. I am not sure what is the intention here, cut the begining or end? And which `[FAIL]` we want to ignore?

Reply:
Accidental change